scripts/tests/test_zkai_relabeling_benchmark_suite.py

```
@@ -0,0 +1,130 @@
+    def test_rust_adapter_requires_exact_case_coverage(self) -> None:
+        original_run = SUITE.subprocess.run
+
+        class FakeCompleted:
+            returncode = 0
+            stderr = ""
+            stdout = (
+                '{"schema":"agent-step-receipt-rust-verifier-adapter-v1",'
+                '"results":[{"case_id":"baseline","accepted":true,"error":""}]}'
+            )
+
+        def fake_run(*_args, **_kwargs):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `fake_run`

(ANN202)